INSTALLATION.md

```diff
@@ -0,0 +1,156 @@
+# Walkthrough 
+
+This is a walkthrough for the OCP worker on VMs project, that can be found here: [ocp-lab-setu]() https://github.com/oVirt/ocp-lab-setup)
+
```

Comment:
please describe that this is a practical walkthrough for a Assisted Installer on 3 +2 BM + X Virtual 
many other combinations are possible, the default one is 3+10 Virtual only.